docs/source/changelog.rst

```diff
@@ -6,10 +6,12 @@ Changelog
     * Enhancements
     * Fixes
     * Changes
+        * Deprecated feature's get_name function, get_feature_names should be used instead
```

Comment:
This should be the `get_names` not `get_name` function. You can also enclose the function names in double back ticks for code formatting: ``get_feature_names``.